docs/source/resources/index.rst

```diff
@@ -0,0 +1,92 @@
+Resources
+=========
+
+.. contents:: |toctitle|
+    :local:
+
+Background
+----------
+
+Differential privacy: A primer for a non-technical audience
+
+https://salil.seas.harvard.edu/publications/differential-privacy-primer-non-technical-audience
+
+The Algorithmic Foundations of Differential Privacy
+
+https://www.cis.upenn.edu/~aaroth/Papers/privacybook.pdf
+
+Learning About Differential Privacy
+-----------------------------------
+
+Courses & Educational Materials from the Harvard University Privacy Tools Project
+
+https://privacytools.seas.harvard.edu/courses-educational-materials
+
+Designing Access with Differential Privacy
+
+https://admindatahandbook.mit.edu/book/latest/diffpriv.html
+
+Programming Differential Privacy
+
+https://programming-dp.com/notebooks/cover.html
+
+Privacy and Confidentiality Protection Overview
+
+https://www2.census.gov/cac/nac/meetings/2019-05/garfinkel-privacy-confidentiality-protection.pdf
+
+Papers
+------
+
+The Complexity of Differential Privacy
+
+https://privacytools.seas.harvard.edu/files/privacytools/files/complexityprivacy_1.pdf
+
+Calibrating Noise to Sensitivity in Private Data Analysis
+
+(Laplace mechanism)
+
+https://people.csail.mit.edu/asmith/PS/sensitivity-tcc-final.pdf
+
+On Significance of the Least Significant Bits For Differential Privacy
+
+(Snapping mechanism and how floating-point numbers and the laplace mechanism leak)
+
+https://www.microsoft.com/en-us/research/wp-content/uploads/2012/10/lsbs.pdf
+
+Mechanism Design via Differential Privacy
+
+(Exponential mechanism)
+
+https://www.microsoft.com/en-us/research/wp-content/uploads/2016/02/mdviadp.pdf
+
+Differential Privacy on Finite Computers
+
+(Geometric mechanism)
+
+https://arxiv.org/pdf/1709.05396.pdf
+
+Privacy-preserving Statistical Estimation with Optimal Convergence Rates
+
+(Sample/aggregate and quantiles)
+
+https://cs-people.bu.edu/ads22/pubs/2011/stoc194-smith.pdf
+
+Deep Learning with Differential Privacy
+
+(Gradient descent)
+
+https://arxiv.org/pdf/1607.00133.pdf
+
+Rényi Differential Privacy
+
+https://arxiv.org/pdf/1702.07476.pdf
+
+Concentrated Differential Privacy: Simplifications, Extensions, and Lower Bounds
+
```

Comment:
Maybe we should hold off on these resources until we support them?

Reply:
Makes sense. I'm not sure what we support.

Reply:
I think dropping the entries for Renyi and Concentrated DP should be enough.